packages/node_modules/@webex/react-component-utils/src/files.js

```diff
@@ -51,7 +53,8 @@ export function constructFile(file) {
     displayName: file.name,
     fileSize: file.size,
     fileSizePretty: bytesToSize(file.size),
-    mimeType: file.type
+    mimeType: file.type,
+    overThreshold: file.size >= FILE_ATTACHMENT_MAX_SIZE
```

Comment:
Sorry I didn't comment on this earlier, I just understood what you're doing.
Since the file size is already passed as part of the file object, I don't we should add this as a property, as it can be computed dynamically in the component.
Hence my suggestion is to just do the check in the component.